src/index.html

```diff
@@ -4,9 +4,378 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__home-link">
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="header__logo"
+          >
+      </a>
+
+      <nav class="navigation">
+        <menu class="navigation__list">
+          <li class="navigation__item">
+            <a class="navigation__link is-active" href="#apple">Apple</a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#samsung">Samsung</a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#smartphones">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link"
+              href="#Laptops"
+              data-qa="nav-hover"
+            >
+              Laptops & computers
+            </a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#gadjets">Gadgets</a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#photo">Photo</a>
+          </li>
+
+          <li class="navigation__item">
+            <a class="navigation__link" href="#video">Video</a>
+          </li>
+        </menu>
+      </nav>
+      </div>
+    </header>
+
+    <main class="main">
+      <section class="catalog main__catalog">
+        <article class="card catalog__card"
+        data-qa="card" >
```

Comment:
maybe better to write attributes in one line or class and data-qa each on a new line)